docs/en/glossary/glossary.asciidoc

```diff
@@ -551,6 +551,20 @@ nodes handling the user requests.
 endif::cloud-terms[]
 ifdef::elasticsearch-terms[]
 
+[[glossary-recovery]] recovery ::
+The process of updating a shard copy and making it available for queries.
++
+Recovery automatically occurs anytime a shard moves to a different node in the same cluster, including:
+
```

Comment:
Does there have to be a `+` here? I can never tell without building it.

Reply:
Yeah. It's needed for indentation.

Screenshot of the output without the `+`.

<img width="756" alt="recovery-indentation" src="https://user-images.githubusercontent.com/40268737/54708585-0f556b80-4b1a-11e9-99e4-c50f8ef93085.png">